auto3dseg/algorithm_templates/segresnet2d/scripts/algo.py

```diff
@@ -48,7 +49,7 @@ def pre_check_skip_algo(self, skip_bundlegen: bool=False, skip_info: str=''):
                 skip_info = f'2D network is skipped due to median spacing of {spacing}.'
```

Comment:
Can we improve the info check:
```
f'2D network is skipped due to median spacing of {spacing}.'
```
by adding:
```
f"SegresNet2D is skipped due to median spacing of {spacing}, which means the dataset is not highly anisotropic, e.g.(res_z < 3*(res_x + rex_y)/2)."
```

Reply:
ok, I updated that